Tools/cases_generator/generators_common.py

```diff
@@ -357,6 +359,18 @@ def reload_stack(
         self.emit_reload(storage)
         return True
 
+    def instruction_size(self,
+        tkn: Token,
+        tkn_iter: TokenIterator,
+        uop: Uop,
+        storage: Storage,
+        inst: Instruction | None
+    ) -> bool:
+        """Replace the INSTRUCTION_SIZE macro with the size of the current instruction."""
+        size = inst.size if inst else 1 + uop.size
+        self.out.emit(tkn.replaceText(str(size)))
```

Comment:
Does `self.out.emit(f"{size}")` directly work?

Reply:
It does, but it does not insert white space around the instruction size, so you end up with

```int x =2;```
instead of 
```int x = 2;```

If that's acceptable, I can emit the size directly

Reply:
How about `self.out.emit(f" {size}")` then? Ideally, we want to keep this simple. `replaceText` confuses me a little.

Reply:
The thing is sometimes the space should be included and sometimes not, like inside parentheses:
```c
(INSTRUCTION_SIZE + oparg) -> (2 + oparg)
```
Same goes for whitespace that comes after `INSTRUCTION_SIZE`. We could simply do `self.out.emit(f" {size} ")` and call it a day, though we'll end up with some weird spacing in the generated code..

Reply:
Yup okay the latter sounds good. I rather a small bit of weirdness in the generated code than the generator being hard to understand. Thanks!

Reply:
updated :)